packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -632,3 +632,185 @@ describe('masternode', () => {
+      })
+    }
+    await listUnspent()
+    return matchingUTXOs
+  }
+
+  //= ==============================================================================================
+  it('should getBalance >= 100', async () => {
+    return await waitForExpect(async () => {
+      const balance: BigNumber = await client.wallet.getBalance()
+      expect(balance.isGreaterThan(new BigNumber('100'))).toBe(true)
+    })
+  })
+
+  it('should send one address using sendMany', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001 }
+    return await waitForExpect(async () => {
+      const transactionId = await client.wallet.sendMany(amounts)
+      expect(typeof transactionId).toBe('string')
+
+      // generate one block
+      await container.generate(1)
+
+      // check the corresponding UTXO
+      const utxos = await getMatchingUTXO(transactionId, 'mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+      // In this case the we should only have one matching utxo
+      expect(utxos.length).toBe(1)
+      utxos.forEach(utxo => {
+        expect(utxo.address).toBe('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+        expect(utxo.amount.isEqualTo(0.00001)).toBe(true)
+      })
+    })
+  })
+
+  it('should send multiple address using sendMany', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const transactionId = await client.wallet.sendMany(amounts)
+      expect(typeof transactionId).toBe('string')
+
+      // generate one block
+      await container.generate(1)
+
+      // check the corresponding UTXOs
+      const utxos = await getMatchingUTXO(transactionId, 'mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+      // In this case the we should only have one matching utxo
+      expect(utxos.length).toBe(1)
+      utxos.forEach(utxo => {
+        expect(utxo.address).toBe('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+        expect(utxo.amount.isEqualTo(0.00001)).toBe(true)
+      })
+
+      const utxos2 = await getMatchingUTXO(transactionId, 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy')
+      // In this case the we should only have one matching utxo
+      expect(utxos2.length).toBe(1)
+      utxos2.forEach(utxo => {
+        expect(utxo.address).toBe('mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy')
+        expect(utxo.amount.isEqualTo(0.00002)).toBe(true)
+      })
+    })
+  })
+
+  it('should sendMany with comment', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const options: SendManyOptions = {
+        comment: 'test comment'
+      }
+      const transactionId = await client.wallet.sendMany(amounts, [], options)
+
+      expect(typeof transactionId).toBe('string')
+
+      // NOTE(sp): How to test the comment is there and where
```

Comment:
perhaps try `getrawtransaction {txid} 1` to check the info inside

Reply:
it does not return `comment`.
maybe should dig more into c++ side to check where it is stored.